automation/utils.ts

```diff
@@ -0,0 +1,174 @@
+/**
```

Comment:
Note that most of the code in this file was simply copied-and-pasted (refactored) from `automation/build-bin.ts`.